src/OpenIDConnect.jl

```diff
@@ -157,6 +188,11 @@ function flow_get_authorization_code(ctx::OIDCCtx, @nospecialize(query))
 
     code = get(query, "code", get(query, :code, nothing))
     if code !== nothing
+        if haskey(ctx.code_verifiers, state)
+            code_verifier = pop!(ctx.code_verifiers, state)
+            # as space characters are not allowed in code strings, it is safe to conacatenate with ' '
+            code *= " $code_verifier"
```

Comment:
Maybe we should have a new `Code` struct that can hold both `code` and `code_verifier`? There is a small chance that it may break code that assumes the returned code to be a string. But it seems much cleaner.

Reply:
Okay, thanks for explaining. Let's keep it as it is.